content/projects/infrastructure/ircbot.adoc

```diff
@@ -4,223 +4,9 @@ title: "IRC Bot"
-prefix of "jenkins" or any other irrelevant information should be removed.
-
-For example:
-
-[source]
-----
-jenkins-admin: fork example/jenkins-foobar as foobar-plugin
-jenkins-admin: fork https://github.com/somebody/foobar-plugin
-----
-
-*Create a new repository on GitHub*
-
-[source]
-----
-jenkins-admin: Create REPO on github
-jenkins-admin: Create REPO on github for SOMEONE
-----
-
-
-In the latter form, the specified someone will get the commit access to the
-repository right away.
-
-=== Issue Tracker Management
-
-*Create an issue tracker component*
-
-[source]
-----
-jenkins-admin: Create COMPONENT in the issue tracker for USER
-jenkins-admin: Create COMPONENT in the bug database for USER
-----
-
-Where COMPONENT is the subcomponent name in the issue tracker (normally the
-name of the plugin), and USER is the jenkins-ci.org user ID who gets the
-initial notification when a new bug is filed.
-
-*Set default assignee of issue tracker component*
-
-[source]
-----
-jenkins-admin: Set USER as default assignee of COMPONENT1[, COMPONENT2, COMPONENT3...]
-jenkins-admin: Make USER the lead of COMPONENT1[, COMPONENT2, COMPONENT3...]
-----
-
-Where COMPONENTx is the subcomponent name in the issue tracker (normally the
-name of the plugin), and USER is the jenkins-ci.org user ID who gets the
-initial notification when a new bug is filed. One or more components can be
-listed by separating them with commas.
-
-*Remove the default assignee*
-
-[source]
-----
-jenkins-admin: Remove the default assignee of COMPONENT1[, COMPONENT2, COMPONENT3...]
-jenkins-admin: Remove lead of COMPONENT1[, COMPONENT2, COMPONENT3...]
-----
-
-*Rename the component*
-
-[source]
-----
-jenkins-admin: Rename component COMPONENT to NEW_NAME
-----
-
-*Modify the component description*
-
-[source]
-----
-jenkins-admin: Set description of COMPONENT to "DESCRIPTION"
-jenkins-admin: Set the description for component COMPONENT to "DESCRIPTION"
-----
-
-*Remove the component*
-
-The command removes the specified component. All its issues will be moved to COMPONENT2
-
-[source]
-----
-jenkins-admin: Remove component COMPONENT1 and move its issues to COMPONENT2
-jenkins-admin: Delete component COMPONENT1 and move its issues to COMPONENT2
-----
+NOTE: The IRC Bot has been deprecated is no longer in use as the Jenkins Infrastructure team has moved to link:https://app.gitter.im/#/room/#jenkins-infra:matrix.org[Matrix].
+The previous functions available through IRC Bot are now unavailable.
+For GitHub permission requests and other repository management issues, you can submit a ticket to the link:https://github.com/jenkins-infra/helpdesk[Jenkins infra help desk].
+Issue reporting and tracking can be found via the link:https://github.com/jenkins-infra/helpdesk/issues[GitHub issues list].
+The link:https://issues.jenkins.io/projects/JENKINS/issues/[Jenkins JIRA] issue page is also available for reporting any issues on Jenkins core.
```

Comment:
this appears to have nothing to do with this page? 

Its also Jira not JIRA


```suggestion
```

Reply:
Ah, I was trying to account for all issue tracking but that doesn't make sense when this page is for infrastructure specific information. Thanks for pointing that out @timja!